vnext/Shared/tracing/tracing.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+// forward declaration.
+namespace facebook {
+namespace jsi {
+class Runtime;
+}
+} // namespace facebook
+
+namespace facebook {
+namespace react {
+namespace tracing {
+void initializeETW();
+void initializeJSHooks(facebook::jsi::Runtime &runtime);
```

Comment:
Slighlty surprized we need these decls here? Weren't these functions used before?

Reply:
Ah saw you cleaned up the private decl... 
Please ignore this :D